queue_services/entity-filer/devops/vaults.gcp.env

```diff
@@ -32,7 +32,7 @@ NATS_FILER_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_FILER_SUBJECT"
 NATS_ENTITY_EVENT_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_ENTITY_EVENT_SUBJECT"
 NATS_EMAILER_SUBJECT="op://nats/$APP_ENV/nats-emailer/NATS_EMAILER_SUBJECT"
 AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
-TOPIC_NAME="op://gcp-queue/$APP_ENV/filer/TOPIC_NAME"
```

Comment:
we can probably remove the TOPIC_NAME from the filer vault. Thoughts Argus?

Reply:
Let's just keep the `TOPIC_NAME` for each service for now.  I think it was designed this way on purpose for the topic that is being subscribed to.